### PR TITLE
fix(protocol): fix vault sending ether and improve comments

### PR DESCRIPTION
**Pull request by dantaik**

### CODE REVIEW

File: packages/protocol/contracts/tokenvault/ERC20Vault.sol [hunk 1]
```diff
@@ -33,7 +34,12 @@ import { EssentialContract } from "../common/EssentialContract.sol";
  * @custom:security-contact hello@taiko.xyz
  */
 
-contract ERC20Vault is EssentialContract, IERC165Upgradeable {
+contract ERC20Vault is
+    EssentialContract,
```
Comment (dantaik) [inline packages/protocol/contracts/tokenvault/ERC20Vault.sol:38]: NFT vault should do the same thing

**Comment by HiddenHead** [comment]
)

File: packages/protocol/contracts/tokenvault/ERC20Vault.sol [hunk 2]
```diff
@@ -116,93 +104,36 @@ contract ERC20Vault is EssentialContract, IERC165Upgradeable {
...
      * by invoking the message call.
-     *
      * @param opt Option for sending ERC20 tokens.
      */
-
     function sendToken(BridgeTransferOp calldata opt)
         external
         payable
         nonReentrant
-        onlyValidAddresses(opt.destChainId, "erc20_vault", opt.to, opt.token)
```
[the remark this answers is not in the document]
Comment (dantaik) [reply packages/protocol/contracts/tokenvault/ERC20Vault.sol]: Good catch. It's not intended. fixed.

File: packages/protocol/contracts/bridge/README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Let's go deeper into the steps that occur when bridging ETH from srcChain to des
 
 User initiates a bridge transaction with `sendMessage` on the source chain which includes:
 
-- `depositValue`, `callValue`, and `processingFee` -- these must sum to `msg.value`.
+- `callValue` and `processingFee` must sum to `msg.value`.
```
[the remark this answers is not in the document]
Comment (dantaik) [reply packages/protocol/contracts/bridge/README.md:20]: I don't see any change, Dani. Did you forget to push?

**Pull request merged**